composer.json

```diff
@@ -31,6 +31,7 @@
         "doctrine/doctrine-cache-bundle": "~1.0",
         "doctrine/doctrine-fixtures-bundle": "~2.2",
         "doctrine/doctrine-migrations-bundle": "~1.0",
+        "doctrine/doctrine-module": "^1.0",
```

Comment:
Requring this in Sylius sounds like an overkill, I guess I will learn what is needed in the rest of the PR but that's quite big deps to add.